connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/MirrorConnectorsWithCustomForwardingAdminIntegrationTest.java

```diff
@@ -0,0 +1,596 @@
+/*
```

Comment:
There are a few checkstyle failures:
```
[ant:checkstyle] [ERROR] /Users/mickael/github/kafka/connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/MirrorConnectorsWithCustomForwardingAdminIntegrationTest.java:135:59: '{' is not preceded with whitespace. [WhitespaceAround]
[ant:checkstyle] [ERROR] /Users/mickael/github/kafka/connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/MirrorConnectorsWithCustomForwardingAdminIntegrationTest.java:145:63: '{' is not preceded with whitespace. [WhitespaceAround]
[ant:checkstyle] [ERROR] /Users/mickael/github/kafka/connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/MirrorConnectorsWithCustomForwardingAdminIntegrationTest.java:269:13: 'block' child has incorrect indentation level 12, expected level should be one of the following: 16, 20. [Indentation]
[ant:checkstyle] [ERROR] /Users/mickael/github/kafka/connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/MirrorConnectorsWithCustomForwardingAdminIntegrationTest.java:270:13: 'block' child has incorrect indentation level 12, expected level should be one of the following: 16, 20. [Indentation]
[ant:checkstyle] [ERROR] /Users/mickael/github/kafka/connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/MirrorConnectorsWithCustomForwardingAdminIntegrationTest.java:271:13: 'block' child has incorrect indentation level 12, expected level should be one of the following: 16, 20. [Indentation]
[ant:checkstyle] [ERROR] /Users/mickael/github/kafka/connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/MirrorConnectorsWithCustomForwardingAdminIntegrationTest.java:272:13: 'block' child has incorrect indentation level 12, expected level should be one of the following: 16, 20. [Indentation]
[ant:checkstyle] [ERROR] /Users/mickael/github/kafka/connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/MirrorConnectorsWithCustomForwardingAdminIntegrationTest.java:273:9: 'block rcurly' has incorrect indentation level 8, expected level should be one of the following: 12, 16. [Indentation]
[ant:checkstyle] [ERROR] /Users/mickael/github/kafka/connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/MirrorConnectorsWithCustomForwardingAdminIntegrationTest.java:477:13: 'block' child has incorrect indentation level 12, expected level should be one of the following: 16, 20. [Indentation]
[ant:checkstyle] [ERROR] /Users/mickael/github/kafka/connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/MirrorConnectorsWithCustomForwardingAdminIntegrationTest.java:478:9: 'block rcurly' has incorrect indentation level 8, expected level should be one of the following: 12, 16. [Indentation]
[ant:checkstyle] [ERROR] /Users/mickael/github/kafka/connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/MirrorConnectorsWithCustomForwardingAdminIntegrationTest.java:556:18: Name 'DEFAULT_PRODUCE_SEND_DURATION_MS' must match pattern '^[a-z][a-zA-Z0-9]*$'. [LocalVariableName]
```

Reply:
Fixed the checkstyle `./gradlew checkstyleMain checkstyleTest` should now be green